src/test/obj_ctl_arenas/obj_ctl_arenas.c

```diff
@@ -291,6 +291,17 @@ main(int argc, char *argv[])
 		ret = pmemobj_ctl_set(NULL,
 			"heap.arenas_assignment_type", &atype);
 		UT_ASSERTeq(ret, 0);
+	} else if (t == 'b') {
+				size_t narenas = 0;
+		ret = pmemobj_ctl_get(pop,
+			"heap.arenas_default_max", &narenas);
+		UT_ASSERTeq(ret, 0);
+		UT_ASSERTne(narenas, 0);
```

Comment:
how do you know it is 0 at this point?

Reply:
this is ASSERT *not equal*. I know it's non-zero because there must be at least one online CPU.